### PR TITLE
Add postgres properties filter example to documentation & k8s installation notice

### PR DESCRIPTION
**Pull request by KoalaGeo**
# Overview
Function discussed on Gitter missing from the documentation to set with a list of properties the attributes of a postgres to expose/unexpose and to order in HTML output. 

# Related Issue / Discussion
https://gitter.im/geopython/pygeoapi?at=635beb9abad3c73752f906fe

# Contributions and Licensing

(as per https://github.com/geopython/pygeoapi/blob/master/CONTRIBUTING.md#contributions-and-licensing)

- [x] I'd like to contribute [feature X|bugfix Y|docs|something else] to pygeoapi. I confirm that my contributions to pygeoapi will be compatible with the pygeoapi license guidelines at the time of contribution.
- [x] I have already previously agreed to the pygeoapi Contributions and Licensing Guidelines

### CODE REVIEW

File: docs/source/data-publishing/ogcapi-features.rst [hunk 1]
```diff
@@ -240,6 +240,28 @@ This provider has support for the CQL queries as indicated in the Provider table
 .. seealso::
   :ref:`cql` for more details on how to use Common Query Language (CQL) to filter the collection with specific queries.
 
+It is possible to control the order and which properties are exposed/unexposed from a PostGreSQL table using ``properties``, see the example below.
```
Comment (tomkralidis) [inline docs/source/data-publishing/ogcapi-features.rst:243]: Suggest to update text to clarify that `properties` can be applied to any feature (or records) provider that supports.

File: docs/source/installation.rst [hunk 2]
```diff
@@ -67,6 +67,44 @@ Using GitHub Container Registry
 
    docker pull ghcr.io/geopython/pygeoapi:latest   
 
+Kubenetes
```
Comment (tomkralidis) [inline docs/source/installation.rst:70]: Is this file intended as part of this PR?

**Review by tomkralidis — CHANGES_REQUESTED** [review]

File: docs/source/installation.rst (hunk 2 above)
Comment (KoalaGeo) [reply docs/source/installation.rst:70]: Yes - I'll update the title. 

We think there's a bug when running on kubenetes with the connection pooling as each request could go to a separate replica and use a new connection

File: docs/source/data-publishing/ogcapi-features.rst (hunk 1 above)
Comment (KoalaGeo) [reply docs/source/data-publishing/ogcapi-features.rst]: Done - https://github.com/geopython/pygeoapi/pull/1032/commits/e33ca798fe4d9a142e2e6fc4725b24b1fbf0e0f9

File: docs/source/installation.rst (hunk 2 above)
Comment (KoalaGeo) [reply docs/source/installation.rst:70]: We'll solve https://github.com/BritishGeologicalSurvey/pygeoapi/issues/27 asap

**Review by kalxas — APPROVED** [review]
